docs/mkdocs.yml

```diff
@@ -65,6 +65,10 @@ copyright: Copyright &copy; MNE-BIDS-Pipeline authors
 
 extra_css:
     - css/extra.css
+# https://squidfunk.github.io/mkdocs-material/reference/data-tables/
+extra_javascript:
+  - https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js
```

Comment:
We should vendor this file instead of pulling it from the CDN

Reply:
Then again, I think we're not vendoring our emojis either, so …

Reply:
Let's wait to vendor until later, then. Maybe in the end we don't need to?